jest.config.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
-  roots: ['<rootDir>'],
+  roots: ['<rootDir>/src/'],
   moduleFileExtensions: ['js'],
   snapshotSerializers: ['enzyme-to-json/serializer'],
```

Comment:
Do we still need this when there is another same setting in the package.json file?

Reply:
We don't need it here, thanks!